### DISCUSSION TITLE
Wallet connect not working on the Mobile Browsers

### DISCUSSION THREAD

**Discussion by almoapple** [WalletConnect/walletconnect-monorepo#3782] [Developer Support] [answered]
When I select the metamask on the mobile site(such as app.uniswap.org or any other site which use wallectconnect),  it opens the metamsk wallet app, but in the wallet, there is no pop-up for the request to connect, and  same as tokenpocket.
when I visit this site https://lab-walletconnect-modal.pages.dev/with-ethereum-provider, the same as above.

iphone 13
ios: 17.0.3
metamask:v7.8.0(1174)
tokenpocket: 1.8.9

It's normal on Android.

**Comment by boidushya** [comment, marked as answer]
Hey, we have an [open PR](https://github.com/WalletConnect/WalletConnectSwiftV2/pull/1186) for a fix. Once its merged, we will reach out to wallets having this issue, thanks for the report🙏
